CHANGELOG.rst

```diff
@@ -14,6 +14,10 @@ Change Log
 Unreleased
 --------------------
 
+[1.2.0] - 2020-12-24
+--------------------
+* Fixed TypeError that pops up sometimes while communicating with the EMSI API.
```

Comment:
can you please add the linespace under date in the first two entries... i forgot to add in mine and have already merged my PR

Reply:
Done